packages/webapp/src/containers/Finances/NewExpense/AddExpense/styles.module.scss

```diff
@@ -65,6 +65,16 @@
     border-radius: 4px;
     padding: 0 5%;
   }
+  .textArea {
+    font-size: 16px;
+    line-height: 24px;
+    font-family: 'Open Sans', 'SansSerif', serif;
+    height: 37px;
```

Comment:
It looks like `height` is not working. I would add `!important`.
Also, I would try to avoid adding the same styles by doing something like:
```
input {
  width: 60%;
  height: 37px;
}
input, .textArea {
  border: 1px solid #efefef;
  box-sizing: border-box;
  border-radius: 4px;
  padding: 0 5%;
}
...
```
But overall, great job!